Explaination about block label AND "Add Block" Dialog search feature. (#7130)
umbraco/UmbracoDocs: 15/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/block-grid-editor.md

* **Label** - Defines a label for the appearance of the Block in the editor. The label can use AngularJS template-string-syntax to display values of properties.

> WARNING : The label is also used for search in the "Add Block" dialog (while content edition). If you do not define any label, the search will not work. The search does not fall back to name. Not label = no search.

Reviewer: ```suggestion
This label is also used for search in the **Add Block** dialog during content editing. If no label is defined, the block will not be searchable. The search does not fall back to the block’s name.
```
